rpm/topology.spec

```diff
@@ -1,32 +1,58 @@
+%define __python /usr/bin/python3
+%define main_version 1.7.0
+%define cacher_version 0.1
+
 Summary: Client tools for OSG Topology
 Name: topology-client
-Version: 1.4.6
+Version: %{main_version}
 Release: 1%{?dist}
-Source: topology-%{version}.tar.gz
+Source: topology-%{main_version}.tar.gz
 License: Apache 2.0
 BuildArch: noarch
 Url: https://github.com/opensciencegrid/topology/
 Requires: python-gnupg
 Requires: python-requests
 
+
 %description
 Client tools that interact with OSG Topology data
 
+
+%package -n topology-cacher
+Summary: A utility for periodically downloading OSG Topology data
+Version: %{cacher_version}
+
+%description -n topology-cacher
+A utility for periodically downloading OSG Topology data.
+
+
 %prep
-%setup -q -n topology-%{version}
+%setup -q -n topology-%{main_version}
 
 %install
 install -D -m 0755 bin/osg-notify %{buildroot}/%{_bindir}/osg-notify
 install -D -m 0644 src/net_name_addr_utils.py  %{buildroot}/%{python_sitelib}/net_name_addr_utils.py
 install -D -m 0644 src/topology_utils.py %{buildroot}/%{python_sitelib}/topology_utils.py
+install -D -m 0755 src/topology_cacher.py %{buildroot}/%{python_sitelib}/topology_cacher.py
+install -D -m 0644 topology-cacher.cron %{buildroot}/etc/cron.d/topology-cacher.cron
 
 %files
 %{_bindir}/osg-notify
 %{python_sitelib}/net_name_addr_utils.py*
 %{python_sitelib}/topology_utils.py*
+%{python_sitelib}/__pycache__/net_name_addr_utils*
+%{python_sitelib}/__pycache__/topology_utils*
+
+%files -n topology-cacher
+%{python_sitelib}/topology_cacher.py*
+%{python_sitelib}/__pycache__/topology_cacher*
+%config(noreplace) /etc/cron.d/topology-cacher.cron
 
 
 %changelog
+* Wed Aug 04 2021 Mátyás Selmeci <matyas@cs.wisc.edu> 1.7.0
+- Add topology-cacher (SOFTWARE-4704)
+
 * Thu Mar 18 2021 Mátyás Selmeci <matyas@cs.wisc.edu> 1.4.6-1
```

Comment:
that's quite the version bump.

Reply:
Yeah this RPM is weird. It only contains topology-utils (and now this) but has the same version as the topology webapp (which we don't actually put into an RPM).